### DISCUSSION TITLE
Clarifier l'obligation de teledeclaration

### DISCUSSION THREAD

**Discussion by valeriemerle** [betagouv/ma-cantine#1977] [Idées]
Dans la page "espace de mes cantines" changer les informations actuelles dans le bandeau
Du 18 juillet au 16 octobre 2022, participez à la collecte de données 2022 !
Dans votre espace cantine, remplissez votre diagnostic, 2021 et télédéclarez vos données (arrêté ministériel du 14 septembre 2022). L'objectif : établir un bilan  de suivi des mesures EGAlim permettant de mieux piloter cette politique publique par l'administration.

**Comment by valeriemerle** [comment]
Pour augmenter la compréhension de ce qu'il y a à télédeclarer mettre le lien du guide de saisie à au niveau du diagnostic

**Reply by hfroot** [reply]
C'est quoi le lien du guide ?

**Comment by valeriemerle** [comment]
On peut aussi préciser le texte réglementaire à l'appui de la fonction "teledeclaration" : 
Un bilan annuel…L’arreté du 14 septembre 2022 fixe les modalités de transmission par les gestionnaires de restaurants collectifs des données nécessaires à l'établissement de ce bilan statistique annuel.
Nous vous proposons de transmettre les informations de votre diagnostic à la DGAL…
Je déclare sur l’honneur…

**Comment by hfroot** [comment]
Fait https://github.com/betagouv/ma-cantine/commit/f753909482aa51a130702f52558043e02c480d3d
